<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1524</width>
    <height>832</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="prod">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>511</width>
      <height>771</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="baseSize">
     <size>
      <width>510</width>
      <height>0</height>
     </size>
    </property>
    <property name="title">
     <string>상품정보</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>491</width>
       <height>701</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="info_box">
      <item>
       <widget class="QLabel" name="img_label">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="baseSize">
         <size>
          <width>492</width>
          <height>492</height>
         </size>
        </property>
        <property name="text">
         <string>상품 이미지</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="prod__info">
        <property name="text">
         <string>상품 내용</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="detail_info">
        <property name="text">
         <string>상품 상세정보</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>730</y>
       <width>199</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>분석 시작</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="analysis">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>981</width>
      <height>771</height>
     </rect>
    </property>
    <property name="title">
     <string>분석 결과</string>
    </property>
    <widget class="QGroupBox" name="analysis_box">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>20</y>
       <width>221</width>
       <height>741</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>201</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>상품 키워드</string>
      </property>
      <widget class="QListWidget" name="prod_keyword">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>181</width>
         <height>231</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>201</width>
        <height>461</height>
       </rect>
      </property>
      <property name="title">
       <string>키워드가 포함된 리뷰</string>
      </property>
      <widget class="QListWidget" name="keyword_in_review">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>181</width>
         <height>431</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QGroupBox" name="analysis_box2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>351</width>
       <height>741</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>331</width>
        <height>721</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="wordcloud">
         <item>
          <widget class="QLabel" name="wordcloud_img">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="bar_chart_layout">
         <item>
          <widget class="QLabel" name="bar_chart_img">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="pie_chart"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="analysis_box3">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>20</y>
       <width>341</width>
       <height>741</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>19</y>
        <width>321</width>
        <height>711</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="five_rating">
         <property name="title">
          <string>5점 Review Keyword</string>
         </property>
         <widget class="QListWidget" name="five_rating_view">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>301</width>
            <height>111</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="four_rating">
         <property name="title">
          <string>4점 Review Keyword</string>
         </property>
         <widget class="QListWidget" name="four_rating_view">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>301</width>
            <height>111</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="thr_rating">
         <property name="title">
          <string>3점 Review Keyword</string>
         </property>
         <widget class="QListWidget" name="thr_rating_view">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>301</width>
            <height>111</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="two_rating">
         <property name="title">
          <string>2점 Review Keyword</string>
         </property>
         <widget class="QListWidget" name="two_rating_view">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>301</width>
            <height>111</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="one_rating">
         <property name="title">
          <string>1점 Review Keyword</string>
         </property>
         <widget class="QListWidget" name="one_rating_view">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>301</width>
            <height>111</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1524</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>analysis_btn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>771</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>809</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>prod_keyword</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>keywordToreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>698</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>771</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>analysis_btn()</slot>
  <slot>keywordToreview()</slot>
 </slots>
</ui>
